buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -573,7 +573,7 @@ class BeamModulePlugin implements Plugin<Project> {
         google_cloud_datastore_v1_proto_client      : "com.google.cloud.datastore:datastore-v1-proto-client:2.1.3",
         google_cloud_firestore                      : "com.google.cloud:google-cloud-firestore", // google_cloud_platform_libraries_bom sets version
         google_cloud_pubsub                         : "com.google.cloud:google-cloud-pubsub", // google_cloud_platform_libraries_bom sets version
-        google_cloud_pubsublite                     : "com.google.cloud:google-cloud-pubsublite", // google_cloud_platform_libraries_bom sets version
+        google_cloud_pubsublite                     : "com.google.cloud:google-cloud-pubsublite:1.5.0", // google_cloud_platform_libraries_bom sets version
```

Comment:
Is there a `google_cloud_platform_libraries_bom` version that will update this client to 1.5.0 instead?

If not I think we should document that this override should be dropped when one is available

Reply:
Added TODO for me to remove this override once the BOM contains a high enough version

Reply:
We should drop the override once the version of google_cloud_platform_libraries_bom is high enough to include pubsublite 1.5.0. Added comment to this effect